includes/classes/sync/pull.php

```diff
@@ -222,6 +224,12 @@ function ( $meta ) {
 			// Update the GC item status.
 			$this->api->set_item_status( $id, $status );
 		}
+		
+		// Call methods the second time to get post working for the acf fields. 
+		// We can later look for a beter approach to handle this but this works just fine.
```

Comment:
Could we add a little extra to this to explain what happens if we don't do this? Like I understand it's to get the `post` working but why does calling it a second time fix this?